clients/ValkeyClients.md

```diff
@@ -0,0 +1,218 @@
+
+- **iovalkey**
+  - GitHub: [iovalkey](https://github.com/valkey-io/iovalkey)
+  - Installation: `npm install iovalkey`
+  - Description: A robust, performance-focused and full-featured Redis client for Node.js. This is a friendly fork of ioredis after this commit.
+
+<!--- This heading should not be included in the TOC --->
+### Feature Comparison Table
+{: .no_toc }
+
+| Feature                                      | valkey-glide | iovalkey  |
+|----------------------------------------------|:------------:|:---------:|
+| **Read from replica**                        |     Yes      |    Yes    |
+| **Exponential backoff to prevent storm**     |     Yes      |    Yes    |
+| **Valkey version compatibility**             |     7.2      |    7.2    |
+| **PubSub state restoration**                 |     Yes      |    No     |
+| **Cluster Scan**                             |     Yes      |    No     |
+| **Latency-Based Read from Replica**          |     No       |    No     |
+| **Client Side Caching**                      |     No       |    No     |
+
+## Java
+
+- **valkey-glide**
+  - GitHub: [valkey-glide](https://github.com/valkey-io/valkey-glide/tree/main/java)
+  - Installation: Available via Maven and Gradle
+  - Description: An open source Valkey client library that supports Valkey and Redis open source 6.2, 7.0 and 7.2. Valkey GLIDE is designed for reliability, optimized performance, and high-availability, for Valkey and Redis OSS based applications. GLIDE is a multi language client library, written in Rust with programming language bindings, such as Java, node.js and Python.
+
+- **Valkey-Java**
+  - GitHub: [valkey-java](https://github.com/valkey-io/valkey-java)
+  - Installation: Available via Maven and Gradle
+  - Description: valkey-java is Valkey's Java client, derived from Jedis fork, dedicated to maintaining simplicity and high performance.
+
+<!--- This heading should not be included in the TOC --->
+### Feature Comparison Table
+{: .no_toc }
+
+| Feature                                      | valkey-glide | valkey-java |
+|----------------------------------------------|:------------:|:-----------:|
+| **Read from replica**                        |     Yes      |     No      |
+| **Exponential backoff to prevent storm**     |     Yes      |     Yes     |
+| **Valkey version compatibility**             |     7.2      |     7.2     |
+| **Cluster support**                          |     Yes      |     Yes     |
+| **TLS/SSL support**                          |     Yes      |     Yes     |
+| **PubSub state restoration**                 |     Yes      |     No      |
+| **Cluster Scan**                             |     Yes      |     No      |
+| **Latency-Based Read from Replica**          |     No       |     No      |
+| **Client Side Caching**                      |     No       |     No      |
+
+
+## Go
```

Comment:
Should we mention it a side note at the end of client are under development (C#, Go) and already planned (PHP) (different notes types), about ValKey-Glide?

Reply:
I'm not sure. It is not relevant for the reader what are the GLIDE plans.